src/workspacer.FocusIndicator/ConfigContextExtensions.cs

```diff
@@ -1,10 +1,19 @@
-﻿namespace workspacer.FocusIndicator
+﻿using System;
+
+namespace workspacer.FocusIndicator
 {
     public static class ConfigContextExtensions
     {
         public static void AddFocusIndicator(this IConfigContext context, FocusIndicatorPluginConfig config = null)
         {
             context.Plugins.RegisterPlugin(new FocusIndicatorPlugin(config ?? new FocusIndicatorPluginConfig()));
         }
+        
+        public static void AddFocusIndicator(this IConfigContext context, Action<FocusIndicatorPluginConfig> onConfig)
```

Comment:
Any reason for this method to return `void`, instead of the plugin created?

Reply:
I maintained the same result type as the already existing extension method above. I will instead update both method return types.